planning/behavior_velocity_planner/invalid-lanelet-design.md

```diff
@@ -0,0 +1,47 @@
+## Invalid Lanelet
+
+### Role
+
+This module plans the velocity of the related part of the path in case there is an invalid lanelet referring to it.
```

Comment:
The meaning of the `invalid lanelet` is ambiguous for newcomers. Would you add some examples or backgrounds of the `invalid lanelets`? (I know it is for the out of odd though.)

Or we can add a link if there is a description page for the `invalid lanelet`.

Reply:
Thanks for your comment @TakaHoribe san. 
Do you think adding the [discussion link](https://github.com/orgs/autowarefoundation/discussions/3275) will be sufficient for clearing ?

Reply:
@TakaHoribe :
I have added some background and examples in the readme file about invalid lanelets.
Please check and let me know if any modifications needed.